lib/csv2avro/converter.rb

```diff
@@ -35,11 +45,19 @@ def convert
           @writer.write(hash)
         rescue CSV2Avro::SchemaValidationError => e
           error_msg = "L#{row_number}: #{e.errors.join(', ')}"
-          @error_writer.puts(error_msg)
+          @log_writer.info(message: error_msg,
+                           event: Log::Event.new('schema_violation',
+                                            true,
+                                            {
+                                              filename: File.basename(@input_path),
+                                              line_no: row_number,
+                                              schema_violations: e.errors
```

Comment:
I don't think its realistic to log all schema violations error as an array. The problems I see:

 * arrays are much more harder to work downstream, they don't fit into a column easily, they need to be processed, etc.
 * the validation errors can be long and the log output might be huge

I think we want to know if the event happens and also how many times, but I'm not sure if including all violations would work.